parm/wflow/aqm_prep.yaml

```diff
@@ -130,8 +176,17 @@ task_aqm_ics:
   command: '&LOAD_MODULES_RUN_TASK; "aqm_ics" "&HOMEdir;/jobs/JSRW_AQM_ICS"'
   envars:
     <<: *default_vars
-    PREV_CYCLE_DIR: '<cyclestr offset="-{{ workflow.INCR_CYCL_FREQ }}:00:00">&COMIN_DIR;</cyclestr>'
-  join: !cycstr '&LOGDIR;/{{ jobname }}_@Y@m@d@H&LOGEXT;'
+    PREV_CYCLE_DIR:
+      cyclestr:
+        attrs:
+          offset: "-{{ workflow.INCR_CYCL_FREQ }}:00:00"
+        value: '&COMIN_DIR;'
```

Comment:
I do like the simpler version you suggest here, but I'm torn. I don't know how much this is going to annoy ppl, but if it does annoy them later, it will be a heavier lift to make that breaking change.